dhis-2/dhis-services/dhis-service-schema/src/main/java/org/hisp/dhis/schema/descriptors/DataApprovalLevelSchemaDescriptor.java

```diff
@@ -51,10 +51,13 @@ public Schema getSchema()
         schema.setOrder( 1370 );
 
         schema.getAuthorities()
-            .add( new Authority( AuthorityType.CREATE_PUBLIC, Lists.newArrayList( "F_SYSTEM_SETTING" ) ) );
+            .add( new Authority( AuthorityType.CREATE_PUBLIC,
+                Lists.newArrayList( "F_SYSTEM_SETTING", "F_DATA_APPROVAL_LEVEL" ) ) );
         schema.getAuthorities()
-            .add( new Authority( AuthorityType.CREATE_PRIVATE, Lists.newArrayList( "F_SYSTEM_SETTING" ) ) );
-        schema.getAuthorities().add( new Authority( AuthorityType.DELETE, Lists.newArrayList( "F_SYSTEM_SETTING" ) ) );
+            .add( new Authority( AuthorityType.CREATE_PRIVATE,
```

Comment:
Actually we want to remove the need for `F_SYSTEM_SETTING` and only require the data approval level authority @vietnguyen .